e2e/test/E2ETests.csproj

```diff
@@ -16,7 +16,6 @@
     <Compile Remove="provisioning\ProvisioningE2ETests.cs" />
     <Compile Remove="provisioning\ProvisioningServiceClientE2ETests.cs" />
     <Compile Remove="provisioning\ProvisioningCertificateValidationE2ETest.cs" />
-    <Compile Remove="provisioning\ReprovisioningE2ETests.cs" />
```

Comment:
Why remove?

Reply:
same reason, it's been removed in main branch